src/main/java/database/controller/FileDatabaseExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+package database.controller;
+
+import database.exception.BadRequestException;
+import io.javalin.Javalin;
+import io.javalin.http.Context;
+import io.javalin.http.NotFoundResponse;
+
+public class FileDatabaseExceptionHandler {
+
+    public void register(Javalin app) {
+        FileDatabaseExceptionHandler fileDatabaseExceptionHandler = new FileDatabaseExceptionHandler();
+        app.exception(BadRequestException.class, fileDatabaseExceptionHandler::handleBadRequest);
+        app.exception(NotFoundResponse.class, fileDatabaseExceptionHandler::handleNotFound);
+        app.exception(Exception.class, fileDatabaseExceptionHandler::handleInternalServerError);
+    }
+
+    private void handleBadRequest(BadRequestException e, Context ctx) {
+        ctx.status(400);
+        ctx.json(new ErrorResponse("400", "Bad Request", e.getMessage()));
```

Comment:
For status code you can use constants from `io.javalin.http.HttpStatus` class, e.g. `HttpStatus.BAD_REQUEST`.